README.md

```diff
@@ -30,7 +30,6 @@ TorchMultimodal requires Python >= 3.8. The library can be installed with or wit
     ```
     git clone --recursive https://github.com/facebookresearch/multimodal.git torchmultimodal
     cd torchmultimodal
-    pip install -r requirements.txt
 
     python setup.py install
```

Comment:
Since we now read requirements into setuptools, we can `pip install torchmultimodal`?